titus-server-master/src/test/java/com/netflix/titus/master/appscale/service/DefaultAppScaleManagerTest.java

```diff
@@ -298,7 +298,7 @@ public void checkV3LiveStreamTargetUpdates() throws Exception {
                     targetsUpdated.add(targetUpdated);
                     latch.countDown();
                 },
-                e -> log.error("Error in v2 live stream for scalable target update {}", e),
+                e -> log.error("Error in v2 live stream for scalable target update {}", String.valueOf(e)),
```

Comment:
remove `{}` and just pass in the exception

Reply:
Done